--- origsrc/ffmpeg-0.10.6/libavfilter/vf_frei0r.c	2012-10-25 16:21:36.000000000 -0500
+++ src/ffmpeg-0.10.6/libavfilter/vf_frei0r.c	2013-01-09 18:39:50.215776200 -0600
@@ -199,7 +199,7 @@ static void *load_path(AVFilterContext *
 {
     char path[1024];
 
-    snprintf(path, sizeof(path), "%s%s%s", prefix, name, SLIBSUF);
+    snprintf(path, sizeof(path), "%s%s%s", prefix, name, ".so");
     av_log(ctx, AV_LOG_DEBUG, "Looking for frei0r effect in '%s'\n", path);
     return dlopen(path, RTLD_NOW|RTLD_LOCAL);
 }
