--- origsrc/ffmpeg-1.2.3/cmdutils.c	2013-08-26 19:13:44.000000000 -0500
+++ src/ffmpeg-1.2.3/cmdutils.c	2013-09-18 01:00:50.015862600 -0500
@@ -196,7 +196,7 @@ static const OptionDef *find_option(cons
     return po;
 }
 
-#if HAVE_COMMANDLINETOARGVW
+#if HAVE_COMMANDLINETOARGVW && !defined(__CYGWIN__)
 #include <windows.h>
 #include <shellapi.h>
 /* Will be leaked on exit */
@@ -256,7 +256,7 @@ static inline void prepare_app_arguments
 {
     /* nothing to do */
 }
-#endif /* HAVE_COMMANDLINETOARGVW */
+#endif /* HAVE_COMMANDLINETOARGVW && !__CYGWIN__ */
 
 static int write_option(void *optctx, const OptionDef *po, const char *opt,
                         const char *arg)
--- origsrc/ffmpeg-1.2.3/libavutil/random_seed.c	2013-08-10 18:23:28.000000000 -0500
+++ src/ffmpeg-1.2.3/libavutil/random_seed.c	2013-09-18 01:01:40.450747300 -0500
@@ -23,7 +23,7 @@
 #if HAVE_UNISTD_H
 #include <unistd.h>
 #endif
-#if HAVE_CRYPTGENRANDOM
+#if HAVE_CRYPTGENRANDOM && !defined(__CYGWIN__)
 #include <windows.h>
 #include <wincrypt.h>
 #endif
@@ -106,7 +106,7 @@ uint32_t av_get_random_seed(void)
 {
     uint32_t seed;
 
-#if HAVE_CRYPTGENRANDOM
+#if HAVE_CRYPTGENRANDOM && !defined(__CYGWIN__)
     HCRYPTPROV provider;
     if (CryptAcquireContext(&provider, NULL, NULL, PROV_RSA_FULL,
                             CRYPT_VERIFYCONTEXT | CRYPT_SILENT)) {
