--- origsrc/ffmpeg/configure	2010-02-07 05:29:28.000000000 -0600
+++ src/ffmpeg/configure	2010-02-07 05:31:37.414359000 -0600
@@ -2169,11 +2169,12 @@ case $target_os in
         shlibdir_default="$bindir_default"
         SLIBPREF="cyg"
         SLIBSUF=".dll"
-        SLIBNAME_WITH_VERSION='$(SLIBPREF)$(FULLNAME)-$(LIBVERSION)$(SLIBSUF)'
+        SLIBNAME_WITH_VERSION='$(SLIBPREF)$(FULLNAME)-$(LIBMAJOR)$(SLIBSUF)'
         SLIBNAME_WITH_MAJOR='$(SLIBPREF)$(FULLNAME)-$(LIBMAJOR)$(SLIBSUF)'
-        SHFLAGS='-shared -Wl,--enable-auto-image-base'
+        SHFLAGS='-shared -Wl,--enable-auto-image-base -Wl,--out-implib,lib$(NAME)/lib$(NAME).dll.a'
+        SLIB_INSTALL_EXTRA_CMD='mkdir -p "$(LIBDIR)"; install -m 644 lib$(NAME)/lib$(NAME).dll.a $(LIBDIR)/lib$(NAME).dll.a'
+        LDCONFIG=""
         objformat="win32"
-        enable dos_paths
         check_cflags -fno-common
         ;;
     *-dos|freedos|opendos)
--- origsrc/ffmpeg/subdir.mak	2010-02-07 05:29:28.000000000 -0600
+++ src/ffmpeg/subdir.mak	2010-02-07 05:31:37.423359500 -0600
@@ -65,10 +65,12 @@ install-lib$(NAME)-shared: $(SUBDIR)$(SL
 	install -d "$(SHLIBDIR)"
 	install -m 755 $$< "$(SHLIBDIR)/$(SLIBNAME_WITH_VERSION)"
 	$(STRIP) "$(SHLIBDIR)/$(SLIBNAME_WITH_VERSION)"
+ifneq ($(OS), Windows_NT)
 	cd "$(SHLIBDIR)" && \
 		$(LN_S) $(SLIBNAME_WITH_VERSION) $(SLIBNAME_WITH_MAJOR)
 	cd "$(SHLIBDIR)" && \
 		$(LN_S) $(SLIBNAME_WITH_VERSION) $(SLIBNAME)
+endif
 	$(SLIB_INSTALL_EXTRA_CMD)
 
 install-lib$(NAME)-static: $(SUBDIR)$(LIBNAME)
