--- origsrc/ffmpeg/Makefile	2009-03-02 14:05:36.000000000 -0600
+++ src/ffmpeg/Makefile	2009-03-02 14:10:19.031250000 -0600
@@ -142,8 +142,8 @@ install-man: $(MANPAGES)
 	install -m 644 $(MANPAGES) "$(MANDIR)/man1"
 
 install-vhook: videohook
-	install -d "$(SHLIBDIR)/vhook"
-	install -m 755 $(HOOKS) "$(SHLIBDIR)/vhook"
+	install -d "$(LIBDIR)/vhook"
+	install -m 755 $(HOOKS) "$(LIBDIR)/vhook"
 
 uninstall: uninstall-progs uninstall-data uninstall-man uninstall-vhook
 
@@ -158,7 +158,7 @@ uninstall-man:
 
 uninstall-vhook:
 	rm -f $(addprefix "$(SHLIBDIR)/",$(ALLHOOKS_SRCS:.c=$(SLIBSUF)))
-	-rmdir "$(SHLIBDIR)/vhook/"
+	-rmdir "$(LIBDIR)/vhook/"
 
 testclean:
 	rm -rf tests/vsynth1 tests/vsynth2 tests/data tests/asynth1.sw tests/*~
